back/app/models/custom_field.rb

```diff
@@ -280,7 +280,12 @@ def printable_legacy?
     ignore_field_types.exclude? input_type
   end
 
-  def importable?
+  def pdf_importable?
+    ignore_field_types = %w[page date files topic_ids image_files file_upload shapefile_upload point line polygon cosponsor_ids ranking matrix_linear_scale]
+    printable? && ignore_field_types.exclude?(input_type)
+  end
+
+  def xlsx_importable?
```

Comment:
Did a little refactoring of names so that we know which types are importable for xlsx vs pdf